### DISCUSSION TITLE
A way to render multiple pages in the same canvas one below the other, with custom dividers which should not show in final html

### DISCUSSION THREAD

**Discussion by parthrc** [GrapesJS/grapesjs#6058] [Q&A]
Hello people, I was looking for a efficient way to rener multiple pages one below the other in the same canvas.
As far as I know we can only render one page at a time in the canvas, so my idea is to create a custom page wrapper component and then render the every page inside those.
I also want a custom divider to show up in between these pages so I can add more pages at that position.
Feature similar to gamma . app

It would be helpful if you could critique by ide or let me know a better approach, and things I should look out for. Thanks

**Comment by kla-ko** [comment]
What do you mean exactly by "page"?

**Reply by parthrc** [reply]
I mean this https://grapesjs.com/docs/modules/Pages.html#initialization.
Kind of like a section, something which contains other components, a wrapper

**Comment by artf** [comment]
One day it [will be possible](https://github.com/GrapesJS/grapesjs/pull/2524#issue-554457305) 🤞
